packages/injected/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/injected-wallets",
-  "version": "2.2.4",
+  "version": "2.3.0",
```

```suggestion
  "version": "2.3.0-alpha.1",
```
This will allow an alpha package be published for testing purposes

---

Sounds good, I've updated it accordingly!